<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Viewport</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Objetos</string>
    </property>
   </widget>
   <widget class="QLabel" name="mainLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>441</width>
      <height>421</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="status">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>490</y>
      <width>471</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="objectList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionRectVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Load OBJ</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>301</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Controle da window</string>
    </property>
    <widget class="QPushButton" name="zoomMinus">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>16</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rotWindowButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotacionar
Window</string>
     </property>
    </widget>
    <widget class="QPushButton" name="panDownButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>V</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="RestoreButtom">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Restaurar</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="panUpButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>^</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="zoomPlus">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>16</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="panRightButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="panLeftButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Clipping</string>
    </property>
    <widget class="QRadioButton" name="lbCheck">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Liang-Barsky</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="csCheck">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cohen-Sutherland</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>281</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>2D</string>
    </property>
    <widget class="QPushButton" name="newPoint">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>+ Ponto</string>
     </property>
    </widget>
    <widget class="QPushButton" name="newLine">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>+ Linha</string>
     </property>
    </widget>
    <widget class="QPushButton" name="newPoligon">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>+ Pol√≠gono</string>
     </property>
    </widget>
    <widget class="QPushButton" name="newCurve">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>+ Curva</string>
     </property>
    </widget>
    <widget class="QPushButton" name="newBSCurve">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>+ B-Spline</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="transButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Transformar</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>3D</string>
    </property>
    <widget class="QPushButton" name="newPoint3D">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>+ Ponto 3D</string>
     </property>
    </widget>
    <widget class="QPushButton" name="newPoligon3D">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>+ Objeto 3D</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Proje;'ao</string>
    </property>
    <widget class="QSlider" name="perspSlider">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="sliderPosition">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QRadioButton" name="projOrt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>99</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ortogonal</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="projPersp">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>99</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Perspectiva</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Valor de d</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
